wiki/Web-Frontend.md

```diff
@@ -0,0 +1,10 @@
+# Web Frontend
+
+## Purpose
+
+1. Provide high-quality, user-friendly, interfaces for MODiX functionality, particularly where the user experience within the Bot Frontend is lacking
+2. Serve as the primary user interface for MODiX configuration
+
+## Methodology
+
+PENDING
```

Comment:
👀

Reply:
Yeah, this was the page I wanted you to write. Since I know basically nothing about the original design goals and methodologies used.